ios/RnExecutorch/models/BaseModel.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "ETModel.h"
+
+@interface BaseModel : NSObject
+{
+@protected
+    ETModel *module;
```

Comment:
Just a thought, shouldn't this be `model`? `module` seems confusing since we have `ETModule` which is an abstraction over the `ETModel`?

Reply:
I am not sure about it but that's how it was called in ETModel.mm so I wanted to keep it the same here